pyproject.toml

```diff
@@ -2,29 +2,17 @@
 # See LICENSE file for licensing details.
 
 [tool.poetry]
-name = "mongos-k8s"
```

Comment:
why delete all this part?

Reply:
These fields are used for when we are in package mode. But a charm is not a package so these fields are not necessary.

This was a request from Carl as to follow best practices